UndertaleModLib/Models/UndertaleEmbeddedTexture.cs

```diff
@@ -316,12 +316,12 @@ public GMImage Image
         /// <summary>
         /// The width of the texture.
         /// </summary>
```

Comment:
We probably want to document that this value can be -1 again, by adding:
```cs
<remarks>
In the case of an invalid or missing image, this will always be <c>-1</c>.
</remarks>
```
Same for `Height` as well.

Reply:
We should also document `FormatQOI` and `FormatBZ2` to always be `false`. I think pattern matching does the null checks automatically.

Reply:
I've added the comments, let me know if those look alright to you

Reply:
Looks good to me!